src/paginate.ts

```diff
@@ -109,7 +109,7 @@ function resolveOptions(
   const page = resolveNumericOption(options, 'page', DEFAULT_PAGE);
   const limit = resolveNumericOption(options, 'limit', DEFAULT_LIMIT);
   const route = options.route;
-  const paginationType = options.paginationType || PaginationTypeEnum.LIMIT;
+  const paginationType = options.paginationType || PaginationTypeEnum.TAKE_AND_SKIP;
```

Comment:
I think it would be better if take/skip is the default because it works with joins. Unlike limit/offset.

Reply:
Overall it would be best to keep the default as limit as the most common query lanaguage is probably MySQL and with 14K downloads now I wouldn't want to make a quick swap without polls and discussions. Let's keep it as limit for now and if you feel strongly about changing the default to skip perhaps start a discssion (if you can?) or I'll start a discussion about it. Only had a few issues with typeorm + group bys which can be solve with skip but I think the majority would pick limit over skip?

Reply:
yeah I guess that makes sense. Limit is the defacto standard for pagination.